### ISSUE TITLE
When i use yolov5 scripts to predict yolov8 model i am getting abnormal scores, predictions

### ISSUE THREAD

**Issue by ajithvcoder** [craft-zhang/tensorflow-lite-cpp-examples#2]
I trained a custom model and i am using your yolov5 scripts  . The output tensor when i use yolov8 model is not matching with python version should i need to update your yolov5 script for yolov8 ? Also the scores, labels, boxes are going to 10000, 2000 ranges

**Comment by craft-zhang** [comment]
Of course, the project is pretty ugly right now. The support for different versions of yolo is also messy. But thank you very much for your contribution and support.

**Comment by ajithkumarmcw** [comment]
thanks for your reply could you give me any links which would work for float32 model for yolov8

**Comment by craft-zhang** [comment]
currently the scores, labels, and boxes are hard coded in the YOLO::run() method and for the input data format you can ref https://github.com/craft-zhang/tensorflow-lite-cpp-examples/blob/a4b9c0215495995fdafc173b74897f15346eb785/yolov5.cc#L217

**Comment by ajithvcoder** [comment]
okay thanks
